pages/windows/color.md

```diff
@@ -0,0 +1,15 @@
+# color
+
+> Set the console foreground and background colors.
+
+- Set the console colors to the default values:
+
+`color`
+
+- Set the console foreground and background to a specific color:
```

Comment:
So this sets both the foreground and background to the same color ? That will be weird. Since you won't be able to differentiate.

Reply:
You provide it with 2 digits, such as `70` which will have a foreground colour of white on a black background.

Reply:
I guess that would be fine actually. :+1:

Even though it's a bit longer, it shows how the command actually works.